src/libmongoc/src/mongoc/mongoc-cursor.c

```diff
@@ -1734,6 +1734,8 @@ _mongoc_cursor_prepare_getmore_command (mongoc_cursor_t *cursor,
    const char *collection;
    int collection_len;
    int64_t batch_size;
+   bson_iter_t iter;
+   const bson_value_t *comment;
```

Comment:
```suggestion
```

Remove unused declaration. That may fix the debug-compile-with-warnings and compile-tracing tasks.